march_joint_inertia_controller/src/joint_inertia_controller.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Project March.
+#include "../include/march_joint_inertia_controller/joint_inertia_controller.h"
```

Comment:
This should work. Otherwise it would be a relative path and will always look one directory up, which would break it when moving this file.
```suggestion
#include "march_joint_inertia_controller/joint_inertia_controller.h"
```